test/test.js

```diff
@@ -37,4 +37,8 @@ describe('postcss-at2x', () => {
   it('should process image and add background size', (done) => {
     runTest('read-background-size.css', {detectImageSize: true}, done);
   });
+
+  it('should resolve image path with custom function', (done) => {
+    runTest('resolve-image-path.css', {resolveImagePath: (value) => {return path.resolve(process.cwd(), value);}}, done);
+  });
```

Comment:
I would add it, but I’m unable to even create proper snapshot for this test. It constantly says that there are 5 snapshots, although there should be 6 of them because this test was added. Do you have the same issue?